metrics/network/network-metrics-iperf3.sh

```diff
@@ -100,6 +106,48 @@ function iperf3_jitter() {
 	echo "Finish"
 }
 
+
+# Test/measure the bandwidth between the interconnection of a server/client.
+# A container takes the role of server and the host takes the role of client.
+function iperf3_host_container() {
+	local test_name="network host to container"
+
+	# Checks iperf3 tool installed in host
+	local cmds=("iperf3")
+	check_cmds "${cmds[@]}"
+
+	local server_command="mount -t ramfs -o size=20M ramfs /tmp && iperf3 -s -V"
+	local server_address=$(start_server "$image" "$server_command" "$server_extra_args")
+	local init_time=1
+
+	# Verify server IP address
+	if [ -z "$server_address" ];then
+		clean_env
+		die "server: ip address no found"
+	fi
+
+	# This time is in order to wait the container initialization
+	sleep $init_time
```

Comment:
This isn't going to be reliable - can you run a command, or check a file, or `grep` for a log pattern (with a timeout) instead maybe?

Reply:
I agree, if we can find a way to wait/check (even if we have a sleep and then check - possibly in a loop), that would be better. Maybe we can do something with `netstat` or similar to check for the server open socket?